packages/x-data-grid-pro/src/hooks/features/infiniteLoader/gridInfniteLoaderInterface.ts

```diff
@@ -0,0 +1,7 @@
+export interface GridInfiniteLoaderApi {
+  /**
+   * Stores the ref of the element at the bottom of the virtual scroller content that triggers the infinite loading.
+   * @ignore - do not document.
+   */
+  unstable_infiniteLoadingTriggerRef: (node: HTMLElement | null) => void;
```

Comment:
Keep in private API?

Reply:
I've replaced it with a `getInfiniteLoadingTriggerElement` method in the private API and moved the interface to the MIT package for the sake of type safety (`getInfiniteLoadingTriggerElement` is being called in the MIT package's `useGridVirtualScroller`)